packages/ui/Marquee/MarqueeBase.js

```diff
@@ -182,6 +191,15 @@ const MarqueeBase = kind({
 	},
 
 	computed: {
+		'aria-label': ({'aria-label': aria, children, distance, willAnimate}) => {
+			if (aria == null && willAnimate && distance > 0) {
+				return React.Children.map(children, c => typeof c === 'string' && c)
+					.filter(Boolean)
+					.join(' ') || aria;
```

Comment:
Why fallback to `aria` here? It reads a bit odd to me since the purpose of this branch is to override `aria`.

Reply:
I could have gone with `void 0` but that looks :yuck:

Reply:
Added some tests to validate the intentions of this.